code/design_pattern/src/facade/daily/tasks/gym/BenchPress.java

```diff
@@ -0,0 +1,7 @@
+package daily.tasks.gym;
+
+class BenchPress {
+    BenchPress() {
+        System.out.println("Gotta get that ass-chest");
```

Comment:
Hey, I appreciate good humor in any form, but some of our readers may not. Please make a more kid-friendly statement!